charts/netbox/values.yaml

```diff
@@ -547,6 +547,31 @@ reportsPersistence:
   accessMode: ReadWriteOnce
   size: 1Gi
   annotations: {}
+## Enable scripts persistence using Persistent Volume Claims
+## ref: https://kubernetes.io/docs/concepts/storage/persistent-volumes/
+## @param scriptsPersistence.enabled Enable reports persistence using PVC
+## @param scriptsPersistence.storageClass PVC Storage Class for volume
+## @param scriptsPersistence.accessMode PVC Access Mode for volume
+## @param scriptsPersistence.size PVC Storage Request for volume
+## @param scriptsPersistence.subPath Existing claim's subPath to use, e.g. "media" (optional)
+## @param scriptsPersistence.existingClaim Name of an existing PVC to be used
+## @param scriptsPersistence.annotations Annotations to add to the PVC
+##
+scriptsPersistence:
+  enabled: false
+  existingClaim: ""
+  subPath: ""
+  ## Data Persistent Volume Storage Class
+  ## If defined, storageClassName: <storageClass>
+  ## If set to "-", storageClassName: "", which disables dynamic provisioning
+  ## If undefined (the default) or set to null, no storageClassName spec is
+  ##   set, choosing the defaserviceMonitorult provisioner.  (gp2 on AWS, standard on
```

Comment:
silly, I know, but I assume `defaserviceMonitorult` is autocorrect going wacky or something, should just be "default" shouldn't it?

Reply:
Oh good catch, it's actually coming from my addition, I think (and then copying and pasting)!

Reply:
we can address this in a separate PR then, not sure it should block merging